makefile

```diff
@@ -461,7 +462,7 @@ MFEM_BUILD_FLAGS = $(MFEM_PICFLAG) $(MFEM_CPPFLAGS) $(MFEM_CXXFLAGS)\
  $(MFEM_TPLFLAGS) $(CONFIG_FILE_DEF)
 
 # Rules for compiling all source files.
-$(OBJECT_FILES): $(BLD)%.o: $(SRC)%.cpp $(CONFIG_MK)
+$(OBJECT_FILES): $(BLD)%.o: $(SRC)%.cpp # $(CONFIG_MK)
```

Comment:
I suspect this is unintentional -- revert it please.

Reply:
Reverted in 9d78e8cc2324196b09cbaeb00715224a0341f205.